kratos/includes/shared_pointers.h

```diff
@@ -55,20 +52,35 @@ unique_ptr<C> make_unique(Args &&...args) {
     return unique_ptr<C>(new C(std::forward<Args>(args)...));
 }
 
-template<typename C, typename...Args>
-shared_ptr<C> static_pointer_cast(Args &&...args) {
-    return std::static_pointer_cast<C>(std::forward<Args>(args)...);
-}
+// template<typename C, typename...Args>
```

Comment:
Better remove?

Reply:
done, i will discuss this with @maceligueta

Reply:
No problem from my side.